Gems/Prefab/PrefabBuilder/PrefabGroupTests.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "PrefabBuilderTests.h"
+#include <IPrefabGroup.h>
+#include <PrefabGroup.h>
+#include <AzCore/Serialization/Json/JsonSystemComponent.h>
+
+namespace UnitTest
+{
+    TEST_F(PrefabBuilderTests, PrefabGroup_FindsRequiredReflection_True)
+    {
+        using namespace AZ::SceneAPI;
+        auto* serializeContext = m_app.GetSerializeContext();
+        ASSERT_NE(nullptr, serializeContext);
+        SceneData::PrefabGroup::Reflect(serializeContext);
+        ASSERT_NE(nullptr, serializeContext->FindClassData(azrtti_typeid<DataTypes::IPrefabGroup>()));
+        ASSERT_NE(nullptr, serializeContext->FindClassData(azrtti_typeid<SceneData::PrefabGroup>()));
+
+        auto findElementWithName = [](const AZ::SerializeContext::ClassData* classData, const char* name)
+        {
+            auto it = AZStd::find_if(classData->m_elements.begin(), classData->m_elements.end(), [name](const auto& element)
+            {
+                return std::strcmp(element.m_name, name) == 0;
```

Comment:
Using plain `strcmp` or add an alias for `AZStd::strcmp`. The `std` namespace shouldn't be directly used.